hal/armv8m/cpu.c

```diff
@@ -182,8 +182,10 @@ char *hal_cpuInfo(char *info)
 	int i;
 	unsigned int cpuinfo;
 
-#ifdef CPU_NRF91
+#if defined(__CPU_NRF9160)
 	cpuinfo = _nrf91_cpuid();
+#elif defined(__CPU_MCXN94X)
+	cpuinfo = _mcxn94x_cpuid();
```

Comment:
How about renaming these to e.g. `platform_cpuid` to remove the `#ifdefs`?

Reply:
I'm not convinced, as this function is arch specific (it returns ARM-m related core revision). I'd rather unify SCB handling instead, perhaps later